README.md

```diff
@@ -18,6 +18,19 @@ To transfer file from local to vm, cd to Project-DevOps and run the following co
 To install Jenkins in the VM shell into your multipass VM and run the installJenkins script: 
 `multipass shell mos-vm`
 
+# 4
+To ensure that jenkins if proper get the ip adress from vm with `ip a` copy paste it into browser of your choice with :8080 at the end 
+`<VM ip adress here>:8080`
+
+# 5 
+Unlock jenkins to install dependencies neccessary for project run :
```

Comment:
Could this be spit out at the end of the script? Making a user go all the way back into the VM and run this seems difficult.